docs/helm-deployment-aws_cloud.md

```diff
@@ -1,9 +1,11 @@
 # Alfresco Content Services Deployment with Helm on AWS
 
+**Hint:** Consider [deploying ACS into EKS](./helm-deployment-aws_eks.md) instead of setting
```

Comment:
Probably better to add a note to the links in https://github.com/Alfresco/acs-deployment/blob/master/README.md#deployment-options that EKS is the recommended way for AWS deployments.

Reply:
1. This note is for people who land here directly, e.g. by deep link from another project
2. Are we sure that EKS is now the recommended way?